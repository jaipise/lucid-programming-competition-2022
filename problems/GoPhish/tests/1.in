3
Olivia Orca
Isaac Irrawaddy
Hector Commerson
6
oliviaOrca
isac iriwaddy
HelctorCommerson
Isac Irrawaddy
Olivia Orcas
Olavia Orcash
