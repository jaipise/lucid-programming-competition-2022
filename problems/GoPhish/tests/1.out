EMPLOYEE
CUSTOMER
BOT
BOT
BOT
CUSTOMER
